packages/shared/src/components/modals/ArticlePostModal.tsx

```diff
@@ -28,15 +29,26 @@ export default function ArticlePostModal({
   ...props
 }: ArticlePostModalProps): ReactElement {
   const { showArticleOnboarding } = useContext(OnboardingContext);
-  const { post, isLoading } = usePostById({ id, isFetchingNextPage });
+  const {
+    post,
+    query: { isLoading, isFetching },
+  } = usePostById({ id });
+  const isPostLoading = isLoading || isFetchingNextPage || isFetching;
```

Comment:
Just for my clarity the `isFetchingNextPage` can happen when the modal is open and you press next?

Reply:
Yes. That's right! 🚀